QUARKUS-1397.md

```diff
@@ -0,0 +1,29 @@
+# Quarkus-1397 - Dev preview for Java 17 in Native mode
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-1397
+
+This test plan cover JDK17 Mandrel dev preview Quarkus integration.
+
+## Related test plans
+
+ - [QUARKUS-728](QUARKUS-728)
+
+## Scope of the testing
+
+Test development will focus on
+- Current native scenarios must work with Java 17
+
+### Impact on testsuites and testing automation:
+
+- Mandrel image based on Java 17 (`quarkus/mandrel-21-jdk17-rhel8`) is in a technical preview stage and is going to be covered by extended platform testing as a subset of testing coverage.
+- Native JDK17 will be covered by the following Jenkins jobs:
+  - rhbq-2.2-extended-platform-testing-trigger
+     - rhbq-2.2-baremetal-ts-native
```

Comment:
No OpenShift? I would consider OpenShift as part of the initial experiments, jobs can be focused on baremetal, but we should at least experiment with OpenShift env too.

Reply:
I have added this job "rhbq-2.7-rhel8-openshift-ts-jvm-ocp-native", but currently, there is no "native" JDK17 pipeline, so needs to be created.